sql/connect/client/jdbc/src/main/scala/org/apache/spark/sql/connect/client/jdbc/SparkConnectDatabaseMetaData.scala

```diff
@@ -112,76 +96,55 @@ class SparkConnectDatabaseMetaData(conn: SparkConnectConnection) extends Databas
 
   override def getExtraNameCharacters: String = ""
 
-  override def supportsAlterTableWithAddColumn: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsAlterTableWithAddColumn: Boolean = true
 
-  override def supportsAlterTableWithDropColumn: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsAlterTableWithDropColumn: Boolean = true
 
-  override def supportsColumnAliasing: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsColumnAliasing: Boolean = true
 
-  override def nullPlusNonNullIsNull: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def nullPlusNonNullIsNull: Boolean = true
 
   override def supportsConvert: Boolean =
     throw new SQLFeatureNotSupportedException
 
   override def supportsConvert(fromType: Int, toType: Int): Boolean =
     throw new SQLFeatureNotSupportedException
 
-  override def supportsTableCorrelationNames: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsTableCorrelationNames: Boolean = true
 
-  override def supportsDifferentTableCorrelationNames: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsDifferentTableCorrelationNames: Boolean = false
 
-  override def supportsExpressionsInOrderBy: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsExpressionsInOrderBy: Boolean = true
 
-  override def supportsOrderByUnrelated: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsOrderByUnrelated: Boolean = true
 
   override def supportsGroupBy: Boolean = true
 
-  override def supportsGroupByUnrelated: Boolean =
-    throw new SQLFeatureNotSupportedException
+  override def supportsGroupByUnrelated: Boolean = true
```

Comment:
Databricks JDBC v1.0.10-oss returns `false` for `supportsOrderByUnrelated` and `supportsGroupByUnrelated`, but obviously, Spark SQL supports those SQL syntax.